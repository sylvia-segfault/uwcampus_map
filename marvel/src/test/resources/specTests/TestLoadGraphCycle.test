# Test if a graph is loaded correctly when it has a cycle

LoadGraph graph cycle.tsv
ListChildren graph A
ListChildren graph B
ListChildren graph C