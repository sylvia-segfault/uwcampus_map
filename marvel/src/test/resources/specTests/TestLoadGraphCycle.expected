# Test if a graph is loaded correctly when it has a cycle

loaded graph graph
the children of A in graph are: B(AB) C(AC)
the children of B in graph are: A(AB) D(BD)
the children of C in graph are: A(AC) D(CD)